bmi_config_files/README.md

```diff
@@ -0,0 +1,46 @@
+# BMI Configuration
+BMI requires a configuration file for each model. The LSTM configuration files contains key value pairs that are used by the BMI to run the model. Below are examples and descriptions of each of such keys, and what type of values are associated.
+
+## Meta data
+These key value pairs contain meta data that is not necessary to run the model, but can be useful to make sure that the model is running as expected.  
+- `time_step: '1 hour'` This can be used by the framework to ensure that the model is called at the right time step.
+- `basin_name: 'Narraguagus River at Cherryfield, Maine'` This is not particularly useful, except for book keeping.
+- `basin_id: '01022500'` This can be used by the framework to combine runoff from different catchments at the correct node.
```

Comment:
Does framework here mean Nextgen and is it using this information?

Reply:
I'm assuming yes, nextgen. And no, not that I'm aware of... Perhaps these configs were included more for "next steps" down the road?

Reply:
Thanks, I'll file this under the "things to clean up" for the next PR.

Reply:
I'll just flag, within the text, as *future work, *still in development, *optional, etc. and call it good.  (Besides, statement *can be used* doesn't necessarily mean that it *is* :) )  Think its okay to keep as long as clearly noted that its not currently being used/read-in by model nor elsewhere.  Believe its just these three items you highlighted ln 6-8.

Reply:
@SnowHydrology - cleaned up this readme a bit, see latest [commit](https://github.com/NOAA-OWP/lstm/pull/28/commits/5f650191c5c4c28a202e4020d0dbdf556ab01a4c).

Reply:
Looks good